consensus/obcpbft/obc-batch.go

```diff
@@ -310,6 +316,11 @@ func (op *obcBatch) processMessage(ocMsg *pb.Message, senderHandle *pb.PeerID) e
 	}
 
 	if req := batchMsg.GetRequest(); req != nil {
+		if !op.deduplicator.IsNew(req) {
```

Comment:
I have a vague feeling that the check shouldn't be at this point, because execution and request proposal are not serialized the same way on all replicas.  Probably test on execute?